src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,12 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        text = text.replaceAll("[^-?0-9]+", " ");
+        List<String> array = Arrays.asList(text.trim().split(" "));
```

Comment:
List usage is redundant here. Feel free to use an array

Reply:
fixed